src/main/java/com/afornalik/ox/ui/UIConsole.java

```diff
@@ -0,0 +1,8 @@
+package com.afornalik.ox.ui;
+
+public class UIConsole implements UIOutputDestination{
+
+    public void say(String dummyString) {
+        System.out.print(dummyString);
```

Comment:
dummyString :D

I will change name for print, but if you prefer say it's ok

Reply:
mistake, it takes name from tests (dummyString), changed say to print also.